bot.py

```diff
@@ -39,7 +39,7 @@ async def on_message(message):
         return
     
     command = message.content.split(" ")[0].replace("\\", "")
-    args = message.content.replace(message.content.split(" ")[0]+" ", "").split(" ")
+    args = message.content.split()[1:}
```

Comment:
did you maybe consider message.content.replace(message.content.replace(message.content.replace(" ".join(message.content.split()[1:})), ""),"")???

Reply:
No